src/sentry/static/sentry/app/views/settings/components/settingsPageHeader.jsx

```diff
@@ -1,54 +1,40 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import styled from 'react-emotion';
+import {Flex} from 'grid-emotion';
 
 class SettingsPageHeading extends React.Component {
   static propTypes = {
-    label: PropTypes.string,
+    title: PropTypes.string,
     action: PropTypes.node,
+    tabs: PropTypes.node,
```

Comment:
nit: I would rename this to something less specific, otherwise I think it kind of implies that there may be some tab-magic going on.